components/measurement/nettests/web_connectivity.js

```diff
@@ -65,6 +65,21 @@ const HttpResponseBodyContainer = styled(Pre)`
   overflow: auto;
 `
 
+const FailureString = ({failure}) => {
+  if (!failure) {
+    return (
+      <div>
+        <Tick size={20} /> null
```

Comment:
Unless the `null` is intentional, will remove it in as I go ahead.

Reply:
Yes the `null` is intentional as it’s to show how we represent no failure in the data